partner_address_street3/views/res_partner.xml

```diff
@@ -25,23 +25,6 @@
                     class="o_address_street"
                 />
             </xpath>
-            <xpath
-                expr="/form//field[@name='child_ids']/kanban/field[@name='street2']"
-                position="after"
-            >
-                <field name="street3" />
-            </xpath>
-        </field>
-    </record>
-
-    <record id="res_partner_kanban_view" model="ir.ui.view">
```

Comment:
Can you add an explanation that the reason for deleting the following code is because the `street2` field no longer exists in the kanban view from [this commit](https://github.com/odoo/odoo/pull/167751/commits/9244df59a62ddf86befe7116e1d04e19d4af1a3b)